doc/adr/0006-protocol-naming-conventions.md

```diff
@@ -0,0 +1,56 @@
+# 6. protocol-naming-conventions
+
+Date: 2021-05-08
+
+## Status
+
+Under Review
+
+## Context
+
+This document describes naming conventions for these protocol components:
+
+* message subject
```

Comment:
Would Cap and make it like..

Subjects (including Reply Subjects)
Stream Names
Consumer Names
Account Names

Reply:
changed